src/hotspot/share/cds/cdsConfig.cpp

```diff
@@ -795,6 +795,7 @@ void CDSConfig::setup_compiler_args() {
     FLAG_SET_ERGO(AOTRecordTraining, false);
     FLAG_SET_ERGO_IF_DEFAULT(AOTReplayTraining, true);
     AOTCodeCache::enable_caching();
+    FLAG_SET_ERGO_IF_DEFAULT(UseAOTCodeLoadThread, true);
```

Comment:
I think you need to it switch off in other cases if it is specified on command line.